extensions/WindowsSandboxExtension/Program.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Windows.AppLifecycle;
+using Microsoft.Windows.DevHome.SDK;
+using Serilog;
+using Windows.ApplicationModel.Activation;
+using WindowsSandboxExtension.Helpers;
+using WindowsSandboxExtension.Providers;
+using WinRT;
+
+namespace WindowsSandboxExtension;
+
+public sealed class Program
+{
+    public static IHost? Host
+    {
+        get; set;
+    }
+
+    [MTAThread]
+    public static void Main([System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArray] string[] args)
+    {
+        // Set up Logging
+        Environment.SetEnvironmentVariable("DEVHOME_LOGS_ROOT", Path.Join(Helpers.Logging.LogFolderRoot, "WindowsSandbox"));
+        var configuration = new ConfigurationBuilder()
+            .AddJsonFile("appsettings_wsb.json")
```

Comment:
I think it would make sense to spell out the places where we have 'wsb',  to `WindowsSandbox` to make things really specific since there are alot of things being added to Dev Home right now